run.sh

```diff
@@ -0,0 +1,5 @@
+# Build Docker image
+docker build -t vaccinewatcher . 
+
+# Run Docker Container
+docker run -i vaccinewatcher:latest 
```

Comment:
I would separate out the run and build commands into a run.sh and a build.sh.

Reply:
I separated these out and placed them in a scripts directory to make it cleaner.